test/gtest-extra.h

```diff
@@ -129,6 +129,7 @@ class suppress_assert {
   ~suppress_assert() {
     _set_invalid_parameter_handler(original_handler_);
     _CrtSetReportMode(_CRT_ASSERT, original_report_mode_);
+    (void)original_report_mode_;
```

Comment:
Maybe use function ```ignore_unused``` https://github.com/fmtlib/fmt/blob/785908ee37f970513099704386f1befca25de625/include/fmt/core.h#L340 to suppress warning?

Example:
https://github.com/fmtlib/fmt/blob/785908ee37f970513099704386f1befca25de625/include/fmt/core.h#L2746-L2748

Reply:
Yes please.